src/Extension/ExtensionInterface.php

```diff
@@ -66,12 +63,7 @@ public function getFunctions();
     /**
      * Returns a list of operators to add to the existing list.
      *
-     * @return array<array> First array of unary operators, second array of binary operators
-     *
-     * @psalm-return array{
-     *     array<string, array{precedence: int, precedence_change?: OperatorPrecedenceChange, class: class-string<AbstractUnary>}>,
-     *     array<string, array{precedence: int, precedence_change?: OperatorPrecedenceChange, class?: class-string<AbstractBinary>, associativity: ExpressionParser::OPERATOR_*}>
-     * }
+     * @return OperatorInterface[]
```

Comment:
in 3.x, the return type should probably be a union of the old and new type, so that static analyzers don't report old code as being totally broken (when using native return types, the type of the BC layer needs to be taken into account)